03.log_and_register_model.py

```
@@ -0,0 +1,130 @@
+# Databricks notebook source
+import yaml
```

Ruff

`yaml` imported but unused

Remove unused import: `yaml`

(F401)